pages/_app.js

```diff
@@ -61,13 +66,34 @@ export default function App({ Component, pageProps }) {
     }
   }
 
+  // submit function to add a new collection
+  function onSubmitCollection(event, collectionColor) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    const newCollection = {
+      ...data,
+      color: event.target.color.value,
+      id: uid(),
+    };
```

Comment:
It seems like this function does not use `collectionColor`, therefore you do not need this parameter.
Additionally, the `color` is already included in `formData`.

```suggestion
  function onSubmitCollection(event) {
    event.preventDefault();

    const formData = new FormData(event.target);
    const data = Object.fromEntries(formData);

    const newCollection = {
      ...data,
      id: uid(),
    };
```